ios/jailed/Tweak.xm

```diff
@@ -0,0 +1,40 @@
+#import "lib/Cycript.framework/Headers/Cycript.h"
+
+%ctor {
+	CYListenServer(31337);
```

Comment:
True, should have a variable to toggle it and then always disable when DEBUG=0. What is the reason for disallowing debug builds? Should be no problem loading a debug binary into a non-debug process. Unless that’s blocked during ipa installation or something?